evap/student/tests/test_views.py

```diff
@@ -305,7 +305,13 @@ def test_xmin_of_all_answers_is_updated(self):
         form[question_id(self.evaluation.general_contribution, self.top_general_questionnaire, self.top_grade_question)] = 2
         form.submit()
 
-        self.assertEqual(set(Answer.__subclasses__()), {RatingAnswerCounter, TextAnswer}, "This requires an update if a new answer type is added")
+        self.assertEqual(
+            set(Answer.__subclasses__()),
+            {RatingAnswerCounter, TextAnswer},
+            "This test requires an update if a new answer type is added. Also, when adding a new question, "
```

Comment:
shouldn't this be "answer" as well?
```suggestion
            "This test requires an update if a new answer type is added. Also, when adding a new answer type, "
```

Reply:
hah, that's true.